package.json

```diff
@@ -10,10 +10,10 @@
     "dev-start": "set NODE_ENV=development && nodemon --exec babel-node src/index.js",
     "build": "babel src --out-dir build",
     "start": "node build/index.js",
-    "undo:seed": "./node_modules/.bin/babel-node ./node_modules/.bin/sequelize db:seed:undo:all",
-    "seed": "./node_modules/.bin/babel-node ./node_modules/.bin/sequelize db:seed:all",
-    "undo:migration": "./node_modules/.bin/babel-node ./node_modules/.bin/sequelize db:migrate:undo:all",
-    "migrate": "./node_modules/.bin/babel-node ./node_modules/.bin/sequelize db:migrate"
+    "undo:seed": "nyc sequelize db:seed:undo:all",
+    "seed": "nyc sequelize db:seed:all",
+    "undo:migration": "nyc sequelize db:migrate:undo:all",
+    "migrate": "nyc sequelize db:migrate"
```

Comment:
why did you remove `./node_modules/.bin/babel-node ./node_modules/.bin/` and replace them with `nyc`?